exercises/practice/parallel-letter-frequency/parallel_letter_frequency_benchmark.rb

```diff
@@ -0,0 +1,29 @@
+require 'minitest/autorun'
+require 'minitest/benchmark'
+require_relative 'sequential_letter_frequency'
+require_relative 'parallel_letter_frequency'
+
+class ParallelLetterFrequencyBenchmarkTest < Minitest::Benchmark
```

Comment:
I don't know if there are other exercises that use benchmarking, but in any case I expect the students to not necessarily know what to do with this file. I would suggest adding some text to the `instructions.append.md` file and some comments to this file to help the student figure out what they can do with this file.

Reply:
The instructions may only apply for those that are working locally, rather than solely on the platform, as well.

But, we can also use https://ruby-doc.org/3.0.6/gems/minitest/Minitest/Benchmark.html as it is included.